src/components/createView.jsx

```diff
@@ -26,14 +27,13 @@ class CreateView extends React.Component {
 
   addRoute (e) {
     e.preventDefault();
-    console.log("clicked!");
     fetch("http://localhost:3060/api/v1/routes", {
       method: "POST",
       body: JSON.stringify({
         "route": {
           "name": this.state.name,
           "description": this.state.description,
-          "author": 1
+          "author": 2
```

Comment:
@AnnaKanska - Did you delete your database and get this call working with the server yet? I don't want to merge this until you its confirmed working.

Reply:
Also - instead of hardcoding author as `2` here, you should use `this.props.authorId` (or whatever you want to call it) and then assign that prop to something like 1 or 2 until you are ready to get it from the login.